app/models/miq_server.rb

```diff
@@ -37,7 +37,7 @@ class MiqServer < ApplicationRecord
   scope :active_miq_servers, -> { where(:status => STATUSES_ACTIVE) }
   scope :recently_active,    -> { where(:last_heartbeat => 10.minutes.ago.utc...Time.now.utc) }
   scope :with_zone_id, ->(zone_id) { where(:zone_id => zone_id) }
-  virtual_delegate :description, :to => :zone, :prefix => true
+  virtual_delegate :description, :to => :zone, :prefix => true, :allow_nil => true, :type => :string
```

Comment:
I didn't see you cal this one out specifically.. is the `allow_nil` meant to come over in this PR?

Reply:
Did you decide on this @kbrock ?

Reply:
ping @kbrock

Reply:
ugh, I'll get rid of it